src/pkgcheck/checks/pkgdir.py

```diff
@@ -130,7 +132,23 @@ def __init__(self, filename, size, **kwargs):
 
     @property
     def desc(self):
-        return f'{self.filename!r} exceeds 20KiB in size; {sizeof_fmt(self.size)} total'
+        return (f'{self.filename!r} exceeds {sizeof_fmt(self.limit)} in size; '
+                f'{sizeof_fmt(self.size)} total')
+
+
+class TotalSizeViolation(results.PackageResult, results.Warning):
+    """The total size of $FILESDIR is too large."""
+
+    limit = 51200
```

Comment:
let's add comment of units here for the size:
```suggestion
    limit = 51200 # bytes = 50KiB
```